### ISSUE TITLE
cfg_kernelsize

### ISSUE THREAD

**Issue by tehreemnaqvi** [yjwu17/BP-for-SpikingNN#2]
Hi,

I am referring to your code. My question is how you are taking cfg_kernel_size?

I didn't get the forward function, how you kept cfg_kernel_size? Below is the snippet of your forward function.

Can you please tell me how did you do that?

torch.zeros(batch_size, **cfg_cnn[0][1], cfg_kernel[0], cfg_kernel[0]**, device=device))

Thanks

**Comment by nguyenhongson1902** [comment]
@tehreemnaqvi, I have the same problems when reading the code

**Comment by QianpengLi577** [comment]
@tehreemnaqvi 
Hi, input image size is [batch_size, 28, 28] 

if conv1 is 3*3 kernel ,4 channel(for example) and padding ,then the c1_mem size is [batch_size, 4, 28, 28]

if conv1 is no padding ,then the c1_mem size is [batch_size, 4 ,26, 26]

thus cfg_cnn[ i ][ 1 ] means the channels of  i^{th} conv layer

cfg_kernel[0] means the picture size of the first layer output

cfg_kernel[1] means the picture size of the second layer output
